selfdrive/updated.py

```diff
@@ -26,66 +26,40 @@
 import datetime
 import subprocess
 import psutil
-from stat import S_ISREG, S_ISDIR, S_ISLNK, S_IMODE, ST_MODE, ST_INO, ST_UID, ST_GID, ST_ATIME, ST_MTIME
 import shutil
 import signal
-from pathlib import Path
 import fcntl
 import threading
-import time
 from cffi import FFI
+from pathlib import Path
+from stat import S_ISREG, S_ISDIR, S_ISLNK, S_IMODE, \
+                 ST_MODE, ST_INO, ST_UID, ST_GID, \
+                 ST_ATIME, ST_MTIME
 
 from common.basedir import BASEDIR
 from common.params import Params
 from selfdrive.swaglog import cloudlog
 
+TEST_IP = "8.8.8.8"
+LOCK_FILE = "/tmp/safe_staging_overlay.lock"
 STAGING_ROOT = "/data/safe_staging"
+if os.getenv("UPDATER_TESTING") is not None:
+  TEST_IP = "localhost"
+  LOCK_FILE = os.getenv("UPDATER_LOCK_FILE", LOCK_FILE)
```

Comment:
Why not always get this from the env variable with the default?

Reply:
Cleaned up, was worried about these getting set outside of testing